Sources/SwiftDocC/Model/ParametersAndReturnValidator.swift

```diff
@@ -0,0 +1,473 @@
+/*
+ This source file is part of the Swift.org open source project
+
+ Copyright (c) 2023 Apple Inc. and the Swift project authors
```

Comment:
It's 2024 now :)

...and same throughout this PR.

Reply:
Like above, this change was made in 2023.

Reply:
If I make additional changes to files I will update the years then.